packages/cli/src/commands/catalog.ts

```diff
@@ -0,0 +1,42 @@
+import { localModulePath } from '../utils';
+import { createServer } from 'http';
+
+export interface CatalogCommandOptions {
+  port: number;
+}
+
+const defaultOptions: CatalogCommandOptions = {
+  port: 4200,
+};
+
+const mergeCatalogDefaultOption = (options: Partial<CatalogCommandOptions>) => {
+  return {
+    ...defaultOptions,
+    ...options,
+  } as CatalogCommandOptions;
+};
+
+const serveCatalog = async (options: CatalogCommandOptions) => {
+  const next = await import(localModulePath('next'));
```

Comment:
Just a question, where is `next` module from ?  node modules ?

Reply:
Yes, it will retrieve next module from the same local root with catalog-server.